README.md

```diff
@@ -341,3 +341,102 @@ try {
     echo 'Exception when calling SellersApi->getMarketplaceParticipations: ', $e->getMessage(), PHP_EOL;
 }
 ```
+
+## Custom Authorization Signer
+There is a possibility to use custom authorization signer.
+This may be useful if you want to have more control on adding "Authorization" signature to request. 
```

Comment:
Please change to:

You may need to do custom operations while signing the API request. You can create a custom authorization signer by  creating an implementation of the `AuthorizationSignerContract` interface and passing it into the `Configuration` constructor array.

Reply:
Will do. Sorry for delayed response. I was off few days.

Reply:
Done.